### DISCUSSION TITLE
obsidian有什么"奇技淫巧"

### DISCUSSION THREAD

**Discussion by makerjackie** [obsidianzh/forum#18] [实用技巧/最佳实践]
在这里分享你的obsidian实践！

**Comment by makerjackie** [comment]
通过`path:文件夹路径`来对图关系进行筛选，`-path:文件夹路径/` 不显示某个文件夹
![image](https://user-images.githubusercontent.com/18050469/102009477-9af14000-3d72-11eb-8d87-79620c3451c1.png)

**Comment by Quorafind** [comment]
在新版 0.10.1 引入了拖拽网页内容或直接复制网页内容到 Obsidian 中，使用 Electron 默认的 Ctrl+shift+V 可以快速粘贴无格式文档
